### PR TITLE
Update eldarion-ajax to v0.16.0 w/ auto-update and metadata

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #
Related issue(s): # #11969 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/eldarion/eldarion-ajax
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
    - Count of watchers: 99
   - Count of stars: 758
   - Count of forks: 164
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] Doesn't already exist / has duplicate issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sashberd** [comment]
This pull request become instead of https://github.com/cdnjs/cdnjs/pull/11969. I tried to fix original PR bu no luck. My git crashed with multiple errors. @sufuf3 all your reviews are here. FYR

File: ajax/libs/eldarion-ajax/package.json [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+  "license": "BSD-3-Clause",
+  "author": "Patrick Altman",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/eldarion/eldarion-ajax.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/eldarion-ajax.min.js"
```
Comment (extend1994) [inline ajax/libs/eldarion-ajax/package.json]: How about using `js/+(eldarion-ajax|polyfills)*` to include source files.
See https://github.com/eldarion/eldarion-ajax/blob/0.16.0/build.sh.

File: ajax/libs/eldarion-ajax/package.json [hunk 2]
```diff
@@ -1,16 +1,27 @@
 {
   "name": "eldarion-ajax",
-  "filename": "eldarion-ajax.min.js",
-  "version": "0.12.0",
+  "filename": "js/eldarion-ajax.min.js",
+  "version": "0.16.0",
   "description": "a library for adding declarative ajax functionality to your website",
-  "homepage": "https://github.com/eldarion/eldarion-ajax/",
   "keywords": [
```
Comment (extend1994) [inline ajax/libs/eldarion-ajax/package.json]: Would be nice if adding some keywords.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 Ping

**Comment by sashberd** [comment]
@extend1994 Sorry, my mistake. FYR

**Comment by extend1994** [comment]
Wondering the PR title is appropriate?

**Pull request merged**